### DISCUSSION TITLE
custom gf wont load

### DISCUSSION THREAD

**Discussion by ATTSTZ** [KadeArchive/Kade-Engine#578] [Q&A]
![1](https://user-images.githubusercontent.com/84761746/119442905-dac14800-bd52-11eb-98f2-2465252d0f3f.PNG)
![2](https://user-images.githubusercontent.com/84761746/119442911-dc8b0b80-bd52-11eb-9a3c-c28f32041c09.PNG)
![3](https://user-images.githubusercontent.com/84761746/119442913-dd23a200-bd52-11eb-8952-6709f3978e57.PNG)
![4](https://user-images.githubusercontent.com/84761746/119442915-ddbc3880-bd52-11eb-9648-41e30f8a6f2f.PNG)
![5](https://user-images.githubusercontent.com/84761746/119442918-de54cf00-bd52-11eb-98bf-e80b3e763b85.PNG)
![6](https://user-images.githubusercontent.com/84761746/119442919-deed6580-bd52-11eb-98a2-dfaaf746f677.PNG)

when the custom gf is selected it loads the normal gf, how do i load the custom one

**Comment by pepperly-art** [comment]
in Playstate.hx there's a section like this:

![image](https://user-images.githubusercontent.com/82689031/119539403-21667080-bd5a-11eb-83cb-458a11f9e09b.png)

add a case for her there.

**Comment by Megaman9009** [comment]
Did you figure it out?
